elixir.json

```diff
@@ -1,6 +0,0 @@
-{
-    "bowerDir": "vendor/bower_components",
```

Comment:
We are not using it anymore, I think that key doesn't even exists on elixir config a lot time ago lol